README.md

```diff
@@ -303,24 +303,26 @@ Have a look at [the milestones](https://github.com/kubernetes-sigs/external-dns/
 
 ## Contributing
 
-We encourage you to get involved with ExternalDNS, as users, contributors or as new maintainers that can take over some parts like different providers and help with code reviews.
+Are you interested in contributing to external-dns? We, the maintainers and community, would love your
+suggestions, contributions, and help! Also, the maintainers can be contacted at any time to learn more
+about how to get involved.
 
-Providers which currently need maintainers:
+We also encourage ALL active community participants to act as if they are maintainers, even if you don't have
+"official" write permissions. This is a community effort, we are here to serve the Kubernetes community. If you
+have an active interest and you want to get involved, you have real power! Don't assume that the only people who
+can get things done around here are the "maintainers". We also would love to add more "official" maintainers, so
+show us what you can do!
 
-* Azure
-* Cloudflare
-* Digital Ocean
-* Google Cloud Platform
+The external-dns project is currently in need of maintainers for specific DNS providers. Ideally each provider
+would have at least two maintainers. It would be nice if the maintainers run the provider in production, but it
+is not strictly required. Provider listed [here](https://github.com/kubernetes-sigs/external-dns#status-of-providers)
+that do not have a maintainer listed are in need of assistance.
 
-Any provider should have at least one maintainer. It would be nice if you run it in production, but it is not required.
-You should check changes and make sure your provider is working correctly.
-
-It would be also great to have an automated end-to-end test for different cloud providers, so help from Kubernetes maintainers and their idea on how this can be done would be valuable.
+The external-dns project is also in need of automated end-to-end tests for different DNS providers. Any help from the
+Kubernetes community and ideas on how this can be accomplished would be valuable.
```

Comment:
@seanmalloy can we drop this entirely and use issues to signal what we need instead of the readme?

Reply:
@Raffo fixed in 4ffb6156650fb547c7cc67b20b6cea9059bd6d02

Reply:
See #1770